tests/osaka/eip7692_eof_v1/gas_test.py

```diff
@@ -104,27 +105,33 @@ def gas_test(
             + (Op.DUP3 + Op.SWAP1 + Op.SUB + Op.PUSH2(slot_warm_gas) + Op.SSTORE)
             # store cold gas: DUP2 is the gas of the baseline gas run
             + (Op.DUP2 + Op.SWAP1 + Op.SUB + Op.PUSH2(slot_cold_gas) + Op.SSTORE)
-            # oog gas run:
-            # - DUP7 is the gas of the baseline gas run, after other CALL args were pushed
-            # - subtract the gas charged by the harness
-            # - add warm gas charged by the subject
-            # - subtract `oog_difference` to cause OOG exception (1 by default)
-            + Op.SSTORE(
-                slot_oog_call_result,
-                Op.CALL(
-                    gas=Op.ADD(warm_gas - gas_single_gas_run - oog_difference, Op.DUP7),
-                    address=address_subject,
-                ),
-            )
-            # sanity gas run: not subtracting 1 to see if enough gas makes the call succeed
-            + Op.SSTORE(
-                slot_sanity_call_result,
-                Op.CALL(
-                    gas=Op.ADD(warm_gas - gas_single_gas_run, Op.DUP7),
-                    address=address_subject,
-                ),
+            + (
+                (
+                    # do an oog gas run, unless skipped with `skip_oog_test`:
+                    # - DUP7 is the gas of the baseline gas run, after other CALL args were pushed
+                    # - subtract the gas charged by the harness
+                    # - add warm gas charged by the subject
+                    # - subtract `oog_difference` to cause OOG exception (1 by default)
+                    Op.SSTORE(
+                        slot_oog_call_result,
+                        Op.CALL(
+                            gas=Op.ADD(warm_gas - gas_single_gas_run - oog_difference, Op.DUP7),
+                            address=address_subject,
+                        ),
+                    )
+                    # sanity gas run: not subtracting 1 to see if enough gas makes the call succeed
+                    + Op.SSTORE(
+                        slot_sanity_call_result,
+                        Op.CALL(
+                            gas=Op.ADD(warm_gas - gas_single_gas_run, Op.DUP7),
+                            address=address_subject,
+                        ),
+                    )
+                    + Op.STOP
+                )
+                if not skip_oog_test
```

Comment:
Style question:  Would 
```
Op.STOP if skip_oog_test
else <rest of code>
```
Be a better construction?  Do we want the longer clause first or the clause triggered by the flag first?
Alternately, could the flag be `out_of_gas_testing: bool = True` instead of `skip_oog_test` to appease both?

Reply:
+1 for the `out_of_gas_testing` flag.